sklearn/model_selection/tests/test_search.py

```diff
@@ -359,14 +360,17 @@ def test_return_train_score_warn():
     estimators = [GridSearchCV(LinearSVC(random_state=0), grid, iid=False),
                   RandomizedSearchCV(LinearSVC(random_state=0), grid,
                                      n_iter=2, iid=False)]
-
+    msg_nsplit = ("The default value of n_splits=3 is deprecated in "
```

Comment:
I don't like the message here. It would be better if you can refer to merged PRs so that users can get similar messages for similar issues.